static/app/utils/profiling/spanChart.tsx

```diff
@@ -2,6 +2,32 @@ import {Rect} from 'sentry/utils/profiling/gl/utils';
 import {SpanTree} from 'sentry/utils/profiling/spanTree';
 import {makeFormatter} from 'sentry/utils/profiling/units/units';
 
+import {Profile} from './profile/profile';
+
+function msToUnit(unit: Profile['unit']): (value: number) => number {
```

Comment:
feels like we already have a function for this somewhere 🤔

Reply:
Oh yeah, I just wrapped it into a make call so we dont always have to pass from/to params